hamming/test_hamming.py

```diff
@@ -0,0 +1,33 @@
+from hamming import hamming
+
+
+def test_unequal_dna_strand_lengths():
```

Comment:
Based on my suggestion on raising an exception for this case, you can actually test that an exception was raised.